dom.bs

```diff
@@ -939,6 +946,34 @@ when something has occurred.
 <a for=/>list</a> of zero or more <a>event listeners</a>). It is initially the empty list.
 <!-- Intentionally not exported. -->
 
+<p>Each {{EventTarget}} object has an associated <dfn for=EventTarget>attached internals</dfn> (null
+or an {{EventTargetInternals}} object), initially null.
+<!-- Intentionally not exported. -->
+
+<p>When a <dfn export for=Event id=concept-eventtarget-constructor>constructor</dfn> of the {{EventTarget}}
+interface, or of an interface that inherits from the {{EventTarget}} interface, is invoked, these steps
+must be run, given the arguments <var>eventTargetCallback</var>:
+
+<ol>
+ <li><p>Let <var>eventTarget</var> be the result of creating a new object using this interface.
```

Comment:
With the new-style constructor steps, creation already happened. You just need to manipulate "this".